spec.html

```diff
@@ -32118,11 +32204,25 @@ <h1>
         </h1>
         <dl class="header">
           <dt>description</dt>
-          <dd>It converts _t_ from UTC to local time.</dd>
+          <dd>
+            It converts _t_ from UTC to local time.
+            The local political rules for standard time and daylight saving time in effect at _t_ should be used to determine the result in the way specified in this section.
+          </dd>
         </dl>
         <emu-alg>
-          1. Return _t_ + LocalTZA(_t_, *true*).
+          1. Let _localTimeZone_ be DefaultTimeZone().
+          1. Let _offsetParseResult_ be Completion(ParseTimeZoneOffsetString(_localTimeZone_)).
+          1. If _offsetParseResult_ is a normal completion, then
+            1. Let _offsetNs_ be _offsetParseResult_.[[Value]].
+          1. Else,
+            1. Let _offsetNs_ be GetNamedTimeZoneOffsetNanoseconds(_localTimeZone_, ℤ(ℝ(_t_) &times; 10<sup>6</sup>)).
+          1. Let _offsetMs_ be truncate(_offsetNs_ / 10<sup>6</sup>).
+          1. Return _t_ + 𝔽(_offsetMs_).
         </emu-alg>
+        <p>If political rules for the local time _t_ are not available within the implementation, the result is equal to _t_ because DefaultTimeZone returns *"UTC"* and GetNamedTimeZoneOffsetNanoseconds returns 0.</p>
```

Comment:
```suggestion
        <p>If political rules for the local time _t_ are not available within the implementation, the result is _t_ because DefaultTimeZone returns *"UTC"* and GetNamedTimeZoneOffsetNanoseconds returns 0.</p>
```

Reply:
@ptomato I don't think you made this change.

Reply:
I realize now there were two instances of it, I had changed one but not the other.